skedulr/app/controllers/shifts_controller.rb

```diff
@@ -2,20 +2,34 @@ class ShiftsController < ApplicationController
   include ShiftsHelper
 
   def index
-    @first_name = current_employee.first_name
-    @employee = current_employee
+    if session[:manager]
```

Comment:
I've been using this so that managers can view their employees' shifts. It is necessary to determine whose shifts are being shown. If we just used current_employee then we'd only ever see the manager's shifts